include/sequence.h

```diff
@@ -161,6 +159,53 @@ typedef enum {
     /* 0x7 */ CHANNEL_IO_PORT_7
 } ChannelIOPort;
 
+typedef enum {
+    /* 0 */ VOL_SCALE_INDEX_BGM_MAIN,
+    /* 1 */ VOL_SCALE_INDEX_FANFARE,
+    /* 2 */ VOL_SCALE_INDEX_SFX,
+    /* 3 */ VOL_SCALE_INDEX_BGM_SUB,
+    /* 4 */ VOL_SCALE_INDEX_MAX
+} VolumeScaleIndex; // May be worth using SequencePlayerId instead
+
+typedef struct {
+    /* 0x00 */ f32 volCur;
+    /* 0x04 */ f32 volTarget;
+    /* 0x08 */ f32 volVelocity;
+    /* 0x0C */ u16 volDuration;
+    /* 0x10 */ f32 freqScaleCur;
+    /* 0x14 */ f32 freqScaleTarget;
+    /* 0x18 */ f32 freqScaleVelocity;
+    /* 0x1C */ u16 freqScaleDuration;
+} ActiveSequencesChannelData; // size = 0x20
+
+typedef struct {
+    /* 0x000 */ f32 volCur;
+    /* 0x004 */ f32 volTarget;
+    /* 0x008 */ f32 volVelocity;
+    /* 0x00C */ u16 volDuration;
+    /* 0x00E */ u8 volScales[VOL_SCALE_INDEX_MAX];
+    /* 0x012 */ u8 volFadeTimer;
+    /* 0x013 */ u8 fadeVolUpdate;
+    /* 0x014 */ u32 tempoCmd;
+    /* 0x018 */ u16 tempoOriginal; // stores the original tempo before modifying it (to reset back to)
+    /* 0x01C */ f32 tempoCur;
+    /* 0x020 */ f32 tempoTarget;
+    /* 0x024 */ f32 tempoVelocity;
+    /* 0x028 */ u16 tempoDuration;
+    /* 0x02C */ u32 setupCmd[8]; // a queue of cmds to execute once the player is disabled
+    /* 0x04C */ u8 setupCmdTimer; // only execute setup commands when the timer is at 0.
+    /* 0x04D */ u8 setupCmdNum; // number of setup commands requested once the player is disabled
+    /* 0x04E */ u8 setupFadeTimer;
+    /* 0x050 */ ActiveSequencesChannelData channelData[16];
+    /* 0x250 */ u16 freqScaleChannelFlags;
+    /* 0x252 */ u16 volChannelFlags;
+    /* 0x254 */ u16 seqId; // active seqId currently playing. Resets when sequence stops
+    /* 0x256 */ u16 prevSeqId; // last seqId played on a player. Does not reset when sequence stops
+    /* 0x258 */ u16 channelPortMask;
+    /* 0x25C */ u32 startSeqCmd; // This name comes from MM
+    /* 0x260 */ u8 isWaitingForFonts; // This name comes from MM
+} ActiveSequences; // size = 0x264
```

Comment:
this represents the data for a single active sequence right? the variable itself is an array of 4 so that makes sense as plural, but this should prob just be 1. same for `ActiveSequencesChannelData`